graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/config/DXGraphQLConfig.java

```diff
@@ -131,6 +149,13 @@ public void deleted(String pid) {
         }
     }
 
+    /**
+     * @return true if ENABLE_INTROSPECTION_MODE property is set to true for the default configuration, or false for any other value.
+     * It will be enabled in development mode by default, and disabled in production mode by default if the property is not defined.
+     */
+    public boolean isIntrospectionEnabled() {
+        return (introspectionEnabled == null && SettingsBean.getInstance().isDevelopmentMode()) || introspectionEnabled;
```

Comment:
Isn't there is a NPE if not in dev mode and instrospectionEnabled == null   ??
Mayeb the expected behviour if dev mode override if : 
Return SettingsBean.getInstance().isDevelopmentMode() || (introspectionEnabled != null && introspectionEnabled)